src/themes/teams/components/Chat/chatItemStyles.ts

```diff
@@ -1,25 +1,76 @@
 import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
 import { ChatItemVariables } from './chatItemVariables'
 import { ChatItemProps } from '../../../../components/Chat/ChatItem'
+import { pxToRem } from '../../../../lib'
+import ChatMessage from '../../../../components/Chat/ChatMessage'
+
+const chatMessageClassname = `& .${ChatMessage.className}`
+const chatMessageAuthorClassname = `& .${ChatMessage.className}__author`
+const chatMessageTimestampClassname = `& .${ChatMessage.className}__timestamp`
 
 const chatItemStyles: ComponentSlotStylesInput<ChatItemProps, ChatItemVariables> = {
-  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    position: 'relative',
-    marginTop: v.margin,
-    marginBottom: v.margin,
-  }),
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const { grouped } = p
+    return {
+      position: 'relative',
+      ...((!grouped || grouped === 'start') && { marginTop: pxToRem(16) }),
```

Comment:
nit: I like the `p.group` and `v.margin` pseudo-conventions because they give you an idea on where the variable comes from